ui/inspectors/user.reel/user.html

```diff
@@ -38,9 +38,9 @@
             },
             "bindings": {
                 "context": {"<-": "@owner.context"},
-                "canDelete": {"<-": "!@owner.object.builtin"},
-                "canRevert": {"<-": "(@owner.object.uid == 0) || !@owner.object.builtin"},
-                "canSave": {"<-": "(@owner.object.uid == 0) || !@owner.object.builtin"},
+                "canDelete": {"<-": "!@owner.object.origin.read_only"},
```

Comment:
the existing conditions are still valid, `origin.read_only` is just another condition